.env.example

```diff
@@ -10,9 +10,10 @@ ASERTO_POLICY_ROOT="todoApp"
 # To target an Aserto hosted authorizer, comment out the lines below and uncomment the section
 # at the bottom of this file.
 ASERTO_AUTHORIZER_SERVICE_URL=localhost:8282
-ASERTO_AUTHORIZER_CERT_PATH='$HOME/.config/topaz/certs/grpc-ca.crt'
 ASERTO_DIRECTORY_SERVICE_URL=localhost:9292
-ASERTO_DIRECTORY_GRPC_CERT_PATH='$HOME/.config/topaz/certs/grpc-ca.crt'
+# On Windows, change these to $HOMEPATH\AppData\Local\topaz\certs\grpc-ca.crt
+ASERTO_AUTHORIZER_CERT_PATH=${HOME}/.local/share/topaz/certs/grpc-ca.crt
+ASERTO_DIRECTORY_CERT_PATH=${HOME}/.local/share/topaz/certs/grpc-ca.crt
```

Comment:
I think this should be `ASERTO_DIRECTORY_GRPC_CERT_PATH`. If we want to rename the env var, we should also check where it used and we might also need to update the docs.

Reply:
I think the variable name should be the same in all languages because we generate the same `.env` file regardless of language. The only exception is .NET.

Reply:
@gimmyxd I added a new `ASERTO_GRPC_CERT_PATH` variable that can be used to set the cert path for both authorizer and directory.